presto-mysql/src/test/java/io/prestosql/plugin/mysql/MySqlQueryRunner.java

```diff
@@ -46,42 +45,38 @@ public static QueryRunner createMySqlQueryRunner(TestingMySqlServer server, Map<
             throws Exception
     {
         try {
-            return createMySqlQueryRunner(server.getJdbcUrl(), connectorProperties, tables);
-        }
-        catch (Throwable e) {
-            closeAllSuppress(e, server);
-            throw e;
-        }
-    }
+            DistributedQueryRunner queryRunner = null;
+            try {
+                queryRunner = new DistributedQueryRunner(createSession(), 3);
```

Comment:
move the init before `try` (the catch clause is pointless as long as `queryRunner` is null)